src/screens/HelpScreen.tsx

```diff
@@ -1,11 +1,36 @@
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import React, { Component } from 'react';
+import { Image, StyleSheet, Text, View, Linking, Platform } from 'react-native';
 import baseStyle from '../styles/base';
+import PhoneButton from '../components/PhoneButton';
 
 export default function HelpScreen() {
+  function dialCall() {
```

Comment:
Can you move this onPress handler function into PhoneButton? I think you mentioned you are gonna do this, but wanted to put it here as a reminder.